packages/api/start/routes/permissions.js

```diff
@@ -207,6 +207,49 @@ Route.delete('permissions/:id', 'PermissionController.destroy').middleware([
 	'auth',
 	getMiddlewareRoles([roles.ADMIN]),
 ]);
+/**
+ * @api {delete} /permissions Delete multiple Permissions
+ * @apiGroup Permissions
+ * @apiPermission ADMIN
+ * @apiHeader {String} Authorization Authorization Bearer Token.
+ * @apiHeaderExample {json} Header-Example:
+ *    {
+ *      "Authorization": "Bearer <token>"
+ *    }
+ * @apiParam {String} ids List of permissions IDs.
+ * @apiParamExample  {json} Request sample:
+ *	/permissions?ids=1,2,3
+ * @apiSuccess {Boolean} success Success Flag
+ * @apiSuccessExample {json} Success
+ *    HTTP/1.1 200 OK
+ *    {
+ *		"success":"true"
+ *    }
+ *@apiError (Forbidden 403) {Object} error Error object
+ *@apiError (Forbidden 403) {String} error.error_code Error code
+ *@apiError (Forbidden 403) {String} error.message Error message
+ *@apiErrorExample {json} Unauthorized Access
+ *    HTTP/1.1 403 Forbidden
+ *		{
+ * 			"error": {
+ *   			"error_code": "UNAUTHORIZED_ACCESS",
+ *   			"message":"Você não tem permissão para acessar esse recurso"
+ * 			}
+ *		}
+ *@apiErrorExample {json} Resource Permission was not found
+ *    HTTP/1.1 400 Bad Request
+ *		{
+ * 			"error": {
+ *   			"error_code": "RESOURCE_NOT_FOUND",
+ *   			"message":"The resource Permission was not found"
+ * 			}
+ *		}
+ */
```

Comment:
Após as mudanças incluir nas rotas:
```javascript
*@apiErrorExample {json} Validation Error: Ids Required
 *    HTTP/1.1 400 Bad Request
 *		{
 * 			"error": {
 *   			"error_code": "VALIDATION_ERROR",
 *   			"message": [
 *     				{
 *       				"message": "ids is required.",
 *       				"field": "ids",
 *       				"validation": "required"
 *     				}
 *   			]
 * 			}
 *		}
```
Se o indicative `exists` for usado incluir também:

```javascript
*@apiErrorExample {json} Validation Error: permission Id should exist in permissions
 *    HTTP/1.1 400 Bad Request
 *		{
 * 			"error": {
 *   			"error_code": "VALIDATION_ERROR",
 *   			"message": [
 *     				{
 *       				"message": "The ids.0 should exist in permissions",
 *       				"field": "Ids.0",
 *       				"validation": "exists"
 *     				}
 *   			]
 * 			}
 *		}
 */
```

Reply:
@alexandreadames  esse erro só aparece quando nenhum dos ids da lista existe, por exemplo esse request `?ids=99999,1` retorna 200
```javascript
{
    "error": {
        "error_code": "RESOURCE_DELETED_ERROR",
        "message": "Um erro ocorreu ao tentar excluir o recurso"
    }
}
```

O que sugere? um erro personalizado? "Nenhum dos ids existe no banco"